test/e2e/atlas/deployments_test.go

```diff
@@ -106,7 +110,7 @@ func TestDeployments(t *testing.T) {
 		lines := strings.Split(o, "\n")
 
 		req.Equal(
-			4, // title, content, empty line, empty line
+			3, // title, content, empty line
```

Comment:
is there any reason why we are not just checking for the deployment name in the output? we already check for no error + we check the template in the unit test

This is what I am proposing in https://github.com/mongodb/mongodb-atlas-cli/pull/2257
![Screenshot 2023-09-15 at 11 01 20](https://github.com/mongodb/mongodb-atlas-cli/assets/5663078/df507cae-dcf7-4586-a6c5-8d9427da67f5)

Reply:
sure makes sense